rust/windows-client/src-tauri/src/client/logging.rs

```diff
@@ -72,33 +72,50 @@ pub(crate) async fn clear_logs_inner() -> Result<()> {
 pub(crate) async fn export_logs_inner(ctlr_tx: CtlrTx) -> Result<()> {
     let now = chrono::Local::now();
     let datetime_string = now.format("%Y_%m_%d-%H-%M");
-    let filename = format!("connlib-{datetime_string}.zip");
+    let stem = format!("connlib-{datetime_string}");
```

Comment:
I think we don't need the `stem` here, we can just get the stem from the path with [file_stem](https://doc.rust-lang.org/std/path/struct.Path.html#method.file_stem) in `exports_log_to` and use that and then we need to move less parameters around.

Reply:
I'm using `stem` to name the directory inside the archive, so I don't think it would work. I'll update the doc comment there to explain.

Reply:
Doesn't something like [this](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=74678b4ee2853e35c70d06b6f9ec1a67) work?

Reply:
I don't think it would do what I'm intending. I'm trying to get a zip file like this where our proposed file name ends up inside the zip even if the user doesn't use our proposed name

![image](https://github.com/firezone/firezone/assets/13400041/e00fcb43-4e9e-437c-bf77-e272cccfb537)

Reply:
And then it unzips to a folder with a useful name even if the extractor software doesn't protect against tar bombs

Reply:
I see what you mean, it can be useful to keep the original stem to use the date to prevent the unzip from overwriting a previous directory with the same name